www/new-prebid-limes.uk/default.html

```diff
@@ -0,0 +1,378 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="icon" type="image/svg+xml" href="noun_Swan_3263882.svg">
+  <title>New Prebid Limes | SWAN Demo</title>
+  <link href="bootstrap.min.css" rel="stylesheet">
+  <link href="blog.css" rel="stylesheet">
+  <link href="advert.css" rel="stylesheet">
+
+  <script>
+    var FAILSAFE_TIMEOUT = 10000;
+
+    window.swan_offers = {
+        advert1: JSON.parse({{ .NewOfferIDAsString }})
+    };
+
+    window.adInfo = function(offer) {
+      var owid = "http://cmp.swan-demo.uk/info?owid=" + btoa(JSON.stringify(window.swan_offers[offer]));
```

Comment:
btoa(JSON.stringify(window.swan_offers[offer])) must not be what the endpoint expects, do i need to convert the JSON into a byte array?